solidity/contracts/libs/CctpMessage.sol

```diff
@@ -19,13 +19,40 @@ import {TypedMemView} from "./TypedMemView.sol";
 
 // @dev CCTP Message version 1
 // @dev copied from https://github.com/circlefin/evm-cctp-contracts/blob/release-2025-03-11T143015/src/messages/Message.sol
+
+/**
```

Comment:
Was this taken from a different commit than the one in the comment above (https://github.com/circlefin/evm-cctp-contracts/blob/release-2025-03-11T143015/src/messages/Message.sol)?

Reply:
nope, I just copied the whole implementation